src/reactapp/src/components/InputField.js

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+// Adapted from https://chakra-ui.com/community/recipes/advanced-formik-integration
```

Comment:
I didn't see any license or copyright information on this page, but I still wanted to give credit by mentioning the source of this code.